.github/workflows/keg-build-images.yml

```diff
@@ -0,0 +1,42 @@
+
+name: keg-build-images
+on:
+  push:
+    branches:
+      - ZEN-543-rebuild-workflow
+      - develop
+
+jobs:
+  build-images:
+    name: 'Build Images'
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    env:
+      GIT_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      KEG_CLI_BRANCH: ZEN-543-updates-for-workflow
```

Comment:
I think you missed this one. I see line `33` was updated. Pretty sure this one should be too

Reply:
updated to support workflow_dispatch inputs as discussed